project-bom/pom.xml

```diff
@@ -100,6 +106,13 @@
                 <version>1.9.2</version>
             </dependency>
 
+            <dependency>
+                <artifactId>guava</artifactId>
+                <groupId>com.google.guava</groupId>
+                <!-- TODO check this version -->
+                <version>18.0</version>
```

Comment:
18.0 is the version Selenium 2.45.0 requires.  So, I think you can remove this `TODO`